docs/guide/browser/locators.md

```diff
@@ -709,6 +709,18 @@ Internally, this method calls `.elements` and wraps every element using [`page.e
 
 - [See `locator.elements()`](#elements)
 
+### nth, first, last
```

Comment:
these need to be separate and have examples

they should also come after all locator-returning methods

Reply:
I'll separate them and add examples.

They currently come after `all` which is a locator-returning method of `Locator`. I figured that the `getByXyz` were in their own group at the top because, although they are methods, they are also root locators of `Page` whereas `all`, `nth`, etc are only methods.

Was your goal for me to move `nth`, `first` and `last` to the top, below the `getXyz`, but leave `all` where it is? Or should they remain below `all`? (Or something else? Just LMK.)

Reply:
just move yours bellow `get`. `all` returns an array, not a locator